modules/dev-tools/assets/js/index.js

```diff
@@ -0,0 +1,10 @@
+import Deprecation from './deprecation';
```

Comment:
Why do you need this file? What advantage does it give you that you cannot do by making small changes to `module.js`?

Reply:
Creating an instance of a class should be in a separated file